app/controllers/runtime/organizations_controller.rb

```diff
@@ -214,6 +228,20 @@ def share_domain(guid, domain_guid)
       [HTTP::MOVED_PERMANENTLY, headers, 'Use DELETE /v2/private_domains/:domain_guid']
     end
 
+    get '/v2/organizations/:guid/isolation_segments', :list_isolation_segments
+    def list_isolation_segments(guid)
+      org = find_guid_and_validate_access(:read, guid)
+
+      dataset = IsolationSegmentModel.dataset.where(organizations: org)
+      message = IsolationSegmentsListMessage.from_params(@params)
+
+      [HTTP::OK, MultiJson.dump(Presenters::V3::PaginatedListPresenter.new(
```

Comment:
I've talked some with @SocalNick about this.

By exposing this resource on the v2 API, it is not longer a v3-only object. Using `V3:PaginatedListPresenter` means that you will get v3 pagination even though you are hitting a URL that begins with "v2." If I'm building an API client, this means that I can no longer key off of the api version number to determine what pagination scheme will be returned.

Given that, the two options are:
1) Don't expose isolation segments through the v2 api
2) Build v2 presenters for isolation segments

Since these are still experimental endpoints, this change doesn't have to block merging this PR, but we need to change it before going GA.